helm/ovn-kubernetes/values.yaml

```diff
@@ -83,6 +83,9 @@ global:
   # -- Indicates if ovn-controller should enable/disable the logical flow in-memory cache when processing Southbound database logical flow changes
   # @default -- true
   enableLFlowCache: true
+
+  # -- Configure to use the IPAMClaims CRD feature with ovn-kubernetes, thus granting persistent IPs across restarts / migration for KubeVirt VMs
+  enablePersistentIPs: false
```

Comment:
I think these were not lexicographically ordered, but theme ordered and you added it in the middle of the flow cache theme. No biggie I guess.

Reply:
OK, I'll move it once everything is right.